### PR TITLE
Separating Playground projects for sample

### PR DESCRIPTION
**Pull request by nickrandolph**

### CODE REVIEW

**Comment by nickrandolph** [comment]
@MichaelBTR can you review this PR please

File: TestProjects/Playground/Playground.Droid/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.mvvmcross.playground">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.mvvmcross.playground" android:installLocation="auto">
 	<uses-sdk android:minSdkVersion="15" android:targetSdkVersion="25" />
-	<application android:allowBackup="true" android:icon="@mipmap/icon" android:label="@string/app_name"></application>
```
Comment (ghost) [inline TestProjects/Playground/Playground.Droid/Properties/AndroidManifest.xml:4]: should this icon have been removed?

Comment (nickrandolph) [reply TestProjects/Playground/Playground.Droid/Properties/AndroidManifest.xml:4]: good catch - probably not

File: TestProjects/Playground/Playground.Forms.Droid/Playground.Forms.Droid.csproj [hunk 2]
```diff
@@ -10,14 +10,14 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Playground.Forms.Droid</RootNamespace>
     <AssemblyName>Playground.Forms.Droid</AssemblyName>
-    <TargetFrameworkVersion>v8.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v7.1</TargetFrameworkVersion>
```
Comment (ghost) [inline TestProjects/Playground/Playground.Forms.Droid/Playground.Forms.Droid.csproj:13]: there seems to be some inconsistencies where the csproj has a target framework of 7.1 now but the xamarin forms nuget package targets 8.0

Comment (nickrandolph) [reply TestProjects/Playground/Playground.Forms.Droid/Playground.Forms.Droid.csproj:13]: yeh but this is consistent with all the other droid projects.

File: TestProjects/Playground/Playground.iOS/Playground.iOS.csproj [hunk 3]
```diff
@@ -84,10 +84,10 @@
   </ItemGroup>
   <ItemGroup>
     <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json">
-      <InProject>false</InProject>
```
Comment (ghost) [inline TestProjects/Playground/Playground.iOS/Playground.iOS.csproj:87]: have you found out what these changes do then as they're the same ones that you didn't want to go in last time

**Review by ghost — APPROVED** [review]

File: TestProjects/Playground/Playground.iOS/Playground.iOS.csproj (hunk 3 above)
Comment (nickrandolph) [reply TestProjects/Playground/Playground.iOS/Playground.iOS.csproj:87]: lol yeh good point - I just realised I missed them too
I had to adjust references in the project file, so missed this change

Comment (nickrandolph) [reply TestProjects/Playground/Playground.iOS/Playground.iOS.csproj:87]: Seems to be a VS thing - changes InProject to Visible element

**Pull request merged**